src/components/Graphs/Graphs.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useMemo, useContext } from 'react'
 
-export type Interval = '24hours' | '1month' | '3months' | 'all'
+export type Interval = 'realtime' | '24hours' | '1month' | '3months' | 'all'
```

Comment:
Hmm, should we call this just `10minutes`? All the other intervals are realtime too, just with slower update interval.

Although realtime sounds maybe a bit clearer for end-users. Both are fine, just thinking out loud 🤷

Reply:
Yeah, i guess that's what it basically is 😆 other difference is that the value are filtered to keep the window the same, elsewhere it doesn't really matter... maybe realtime fits there

Reply:
Yeah, true 👍